### PR TITLE
Implement solution for creating documentation as a separate Bamboo plan

### PR DESCRIPTION
**Pull request by rizbihassan**
## Summary of Changes

Addresses [ORCA-523: Implement solution for creating documentation as a separate Bamboo plan](https://bugs.earthdata.nasa.gov/browse/ORCA-523)

## Changes

* updated bamboo spec file and versioning release doc

## Type of change

Please delete options that are not relevant.

- [x ] Bug fix (non-breaking change which fixes an issue)
- [ x] This change requires a documentation update

## How Has This Been Tested?

successfully ran in bamboo and deployed the orca webpage. 
https://ci.earthdata.nasa.gov/browse/ORCA-RDOD-27

## Checklist:

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [ x] Development documentation
- [ x] My changes generate no new warnings
- [x ] I have added tests that prove my fix is effective or that my feature works
    - [x ] Manual tests (outlined above)
- [ x] My code has passed security scanning
    - [ x] Snyk
    - [ x] git-secrets

## What is the current behavior?

deploys the orca webpage without error.

### CODE REVIEW

File: bamboo-specs/bamboo.yaml [hunk 1]
```diff
@@ -255,18 +228,130 @@ version: 2
...
+  - checkout:
+      force-clean-build: 'true'
+      description: Checkout Default Repository
+  - script:
+      interpreter: SHELL
+      scripts:
+      - |-
+        #!/bin/bash
+
+        cat << EOF > tempdockerfile
```
Comment (AndrewDorn) [inline bamboo-specs/bamboo.yaml:270]: Comment on tempdockerfile's origin

File: bamboo-specs/bamboo.yaml [hunk 2]
```diff
@@ -255,18 +228,130 @@ version: 2
...
+  key: ROD
+  name: Release ORCA Documentation
+stages:
+- Release Stage:
+    manual: true
+    final: true
+    jobs:
+    - Release ORCA Documentation
+
+Release ORCA Documentation:
```
Comment (AndrewDorn) [inline bamboo-specs/bamboo.yaml:257]: Comment on why we aren't using the `image` property to set up the container.

**Review by AndrewDorn — APPROVED** [review]

File: bamboo-specs/bamboo.yaml (hunk 2 above)
Comment (rizbihassan) [reply bamboo-specs/bamboo.yaml:257]: using the orca container raises authentication error during npm deploy. See ORCA-467 for details.

**Comment by rizbihassan** [comment]
created "Move bamboo plans into their own files for easier maintenance."
https://bugs.earthdata.nasa.gov/browse/ORCA-591

**Review by bhazuka — APPROVED** [review]

**Pull request merged**